package/opt/hassbian/suites/cloud9.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
+function cloud9-show-short-info {
+  printf "Cloud9 install script for Hassbian.\\n"
+}
+
+function cloud9-show-long-info {
+  printf "Installs Cloud9 IDE onto this system.\\n"
+  printf "Cloud9 IDE is an online integrated development environment. It supports hundreds of programming languages.\\n"
+}
+
+function cloud9-show-copyright-info {
+  printf "This script was originally contributed by Ludeeus <https://github.com/ludeeus>.\\n"
+}
+
+function cloud9-install-package {
+node=$(which node)
+if [ -z "${node}" ]; then #Installing NodeJS if not already installed.
+  printf "Downloading and installing NodeJS...\\n"
+  curl -sL https://deb.nodesource.com/setup_8.x | bash -
```

Comment:
8x is a rather old version and will cause problems with newer tools such as the miio tool used by the Xiaomi vacuum.

Reply:
8 is what we use for homebridge as well, so I need to test that before changing to 9 (the current version).